api/envoy/data/core/v2alpha/local_reply_configuration.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.data.core.v2alpha;
+
+option java_outer_classname = "LocalReplyConfigurationProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.data.core.v2alpha";
+
+// [#protodoc-title: Local reply configuration]
+
+// Configure the local reply of the body type
+message LocalReplyConfiguration {
+  message JsonReply {
```

Comment:
#4705

This issue mentions the need to remove the dependency of RapidJSON. The json conversion uses the method of protobuf to convert, so I defined a protobuf for converting the local reply body.

Reply:
Oh, interesting, I didn't know about that. In that case, my recommendation would be to just instantiate a `ProtobufWkt::Struct` rather than defining a message here.

Reply:
I hope to have a clear description scheme

Reply:
Yeah, Struct would be the canonical representation of JSON in proto.